src/main/java/pl/debuguj/parkingspacessystem/enums/DriverType.java

```diff
@@ -0,0 +1,40 @@
+package pl.debuguj.parkingspacessystem.enums;
+
+import java.math.BigDecimal;
+import java.util.Arrays;
+
+/**
+ * Created by grzesiek on 07.10.17.
+ */
+public enum DriverType {
+    REGULAR("REGULAR", new BigDecimal("2.0"), BigDecimal.ONE),
```

Comment:
To można było zaprojektować w taki sposób, aby te przeliczniki były łatwo konfigurowalne - to brzmi jak coś, co może się zmieniać.